src/lib.rs

```diff
@@ -77,7 +77,7 @@
 //!         "complex32", "complex64", # Complex floating point storage types.
 //!         "f32", "f64", # Floating point storage types.
 //!         "si", "std", # Built-in SI system and std library support.
-//!         "use_serde", # Serde support.
+//!         "serde", # Serde support.
```

Comment:
Could you add a note here and in README.md that the old `use_serde` feature is deprecated and will be removed no earlier than `v0.37.0`.